builds/e2e/nested-e2e.yaml

```diff
@@ -0,0 +1,46 @@
+trigger: none
+pr: none
+
+resources:
+  pipelines:
+  - pipeline: images
+    source: 'Azure-IoT-Edge-Core Build Images'
+    branch: master
+    trigger:
+      branches:
+      - master
+  - pipeline: packages
+    source: 'Azure-IoT-Edge-Core Edgelet Packages'
+    branch: master
```

Comment:
it seems this is optional, will it pick up the triggered branch?

Reply:
btw I think we want to run it explicitly.  Therefore we should use branch argument and get this from pipeline variable.

Reply:
@philipktlin This is structured exactly the same as the new end-to-end test pipeline. It is automatically triggered whenever the 'Build Images' or 'Edgelet Packages' pipeline completes a run based on master. If you run the pipeline manually, it will automatically pick the most recent successful runs of 'Build Images' and 'Edgelet Pipelines' from master, but you can override the defaults with your own build IDs if desired. Does that match what you're thinking?

@yophilav I thought you wanted this pipeline to _only_ run manually. If that's the case, you'll want to remove the `trigger`s from each resource pipeline here.